bin/mail/hoc-2015-mailing/generate-mailing-list-hoc-2015-general

```diff
@@ -93,3 +95,5 @@ puts "#{TEACHERS.count} teachers loaded."
 all.merge! TEACHERS
 p all.count
 export_contacts_to_csv all, "hoc-2015-general.csv"
+
+# rubocop:enable Style/MultilineBlockChain
```

Comment:
Given that there were a bunch of violations to this file, variables names weren't obvious, and this file is "deprecated", I'm blacklisting the cop rather than fixing. Feel welcome to suggest otherwise.